CHANGES.md

```diff
@@ -2,6 +2,8 @@
 
 ### Added
 
+- Add command `delegate-info` to print info needed by external release scripts
```

Comment:
Missing `(#221, @pitag-ha)` at the end of the line

Reply:
Thanks. For some reason I missed that habit.